setup.py

```diff
@@ -47,7 +47,7 @@ def check_dependencies():
                             'clean'))):
         check_dependencies()
 
```

Comment:
I would suggest a more elegant way to alert the user of a missing dependency, currently it looks like this:

```
python setup.py install
Traceback (most recent call last):
  File "setup.py", line 48, in <module>
    check_dependencies()
  File "setup.py", line 35, in check_dependencies
    raise ImportError("Missing dependencies: %s" % missing_deps)
ImportError: Missing dependencies: ['nipype']
```
If you want it to resolve them automatically, you can add `install_requires`. If you want the user to have control over this (manual) then you might have a more elegant fail / alert, or provide a requirements.txt file with instructions to the user to install this first:

```
pip install -r requirements.txt
```
and you can generate this list with pip freeze, and edit the file only if you (for some reason) don't have preference for  version:

```
pip freeze >> requirements.txt
```
Poldracklab probably already has this, but a really easy way to get around this is to give the user a Dockerized pydeface to start with, and then just make sure it has the dependencies! I haven't used this before, but given that it requires fsl or other fat neuroimaging software, you would want to at least mention this, provide in an image (it's going to be big and fat) or check and give the user a nice message if not found.